disposable_email_blocklist.conf

```diff
@@ -799,6 +809,7 @@ ecallheandi.com
 ecolo-online.fr
 edgex.ru
 edinburgh-airporthotels.com
+edu.sg
```

Comment:
This is not a domain, it's a public suffix, which is causing CI to fail here.

Reply:
You are probably correct, this [does not seem like a disposable e-mail address](https://check-mail.org/domain/edu.sg/). My bad, should I add an extra commit removing this item?

Reply:
Yes, please remove it. Thanks!